packages/contracts/test/test-helpers/constants.ts

```diff
@@ -52,3 +27,21 @@ export const L2_TO_L1_MESSAGE_PASSER_OVM_ADDRESS =
 export const CHAIN_ID = 108
 export const ZERO_UINT = '00'.repeat(32)
 export const DEFAULT_FORCE_INCLUSION_PERIOD = 600
+
+export const DEFAULT_UNSAFE_OPCODES = UNSAFE_OPCODES.concat([Opcode.CHAINID])
```

Comment:
Why is `CHAINID` concatenated here? Should we just add `CHAINID` to `UNSAFE_OPCODES`?

Reply:
`UNSAFE_OPCODES` comes from `@eth-optimism/rollup-core`. I didn't know if there was a reason why `CHAINID` wasn't part of `UNSAFE_OPCODES` there, didn't want to touch it.